benchmarks/transport/CMakeLists.txt

```diff
@@ -14,6 +14,9 @@ if (GHEX_USE_UCP)
         if (GHEX_USE_PMIX)
             target_compile_definitions(${_t} PRIVATE USE_PMIX)
         endif()
+        if (GHEX_USE_UCP)
```

Comment:
@boeschf @mbianco We have `GHEX_USE_UCP` and `GHEX_USE_PMIX` defined in `cmake`, while I was using `USE_UCX` and `USE_PMIX` defines in my code. I guess the `GHEX_USE_*` is your preferred naming strategy? If that is the case, then I will simply move all my defines to that scheme.

Reply:
I think `GHEX_` prefix is better to avoid any clashes

Reply:
My approach here would be to collect such things in the `config.h` file and not the commandline. I had a PR for that. Not sure if you guys will want that or not.